File: mario.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    int x = 0;
    int n = 1;
    int height = 0;
    int i = 1;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);
    for (i = 1; i <= height; i++)
    {
        for (x = 0; x < height - i; x++)
        {
            printf(" ");
        }
        for (n = height; n > height - i; n--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int, get_string

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

for i in range(n):
    print(" " * (n-(i+1)), end = "")

    print("#" * (i+1))